### PR TITLE
some improvements on german translation

### PR DESCRIPTION
**Pull request by airbenich**

### CODE REVIEW

File: api/locales/de.json [hunk 1]
```diff
@@ -534,7 +534,7 @@
   "help": "Hilfe",
   "submit": "Absenden",
   "write_comment": "Kommentar schreiben...",
-  "new_item": "Neuer Eintrag",
+  "new_item": "neu",
```
Comment (WellingGuzman) [inline api/locales/de.json]: Having `nue` only is better than "new item"?

Comment (WellingGuzman) [reply api/locales/de.json]: @airbenich ping

**Comment by airbenich** [comment]
„Neuer Eintrag“ breaks the design. German words and sentences are always long compared to english.

**Comment by WellingGuzman** [comment]
Thanks @airbenich! I was just wondering the reason :) that makes sense.

**Comment by rijkvanzanten** [comment]
@WellingGuzman We probably should checkout the design reason btw, I'm pretty sure this problem happens in Dutch as well

**Comment by WellingGuzman** [comment]
@rijkvanzanten Flexible width layouts? (probably same with Spanish too)

**Comment by rijkvanzanten** [comment]
Yes please. I don't know what happened exactly, but some button probably got a `max-width` it shouldn't have. It's not something we go in and fix for everything now, but it's a good thing to remind this issue the next time we add `max-width: $too-small;` to the CSS somewhere :)

**Comment by WellingGuzman** [comment]
Somehow we gotta have a max-width either way, otherwise a text can mess up the layout.

**Pull request merged**